app/components/NextEventInfo/NextEventInfo.tsx

```diff
@@ -22,6 +22,16 @@ const EVENT_TIME_FORMAT = new Intl.DateTimeFormat("en-US", {
   timeZone: "America/Chicago",
 });
 
+function formatDateTime(dateTime: string) {
+  const narrownNonBreakSpace = String.fromCharCode(8239);
+
+  // Discrepency between server and client, see https://github.com/remix-austin/remixaustin-com/issues/83#issuecomment-1450654389
+  return EVENT_TIME_FORMAT.format(new Date(dateTime)).replaceAll(
+    narrownNonBreakSpace,
+    " "
+  );
+}
```

Comment:
Eventually we can pull this into a util file, but I'd rather wait to do that until I see multiple components/functions using it

Reply:
i'd expect there to be a corresponding test for this logic. (also, a nitpick would be to make `narrownNonBreakSpace` a constant outside the function somewhere (maybe in the utils?)

Reply:
I was being lazy, thanks for calling this out. And yeah, definitely could make that a constant since it doesn't need to be reevaluated on each function call. I suppose I can go ahead and pull this out in a util file so I can easily test it.

Reply:
Added a test, if it looks good I'll go ahead and merge.

I wasn't able to figure out a good way to mock Intl.DateTimeFormat to force it to use the narrow non-break space. Didn't feel like it was worth much more time, but if someone has a solution I'm all ears. Otherwise will merge.